docs/contributing/style_guide.rst

```diff
@@ -65,3 +65,40 @@ Encoding
 --------
 Python 3 assumes that files are encoded with UTF-8, so it is not necessary to declare this in the 
 file.
+
+.. _error-handling:
+
+Error Handling
+--------------
+
+Errors in Tasks
+***************
+
+All uncaught exceptions in a task are treated as fatal exceptions. The task is then marked as
+failed. The error traceback, description, and code are returned to the user under the 'error'
+attribute of the task object.
```

Comment:
Suggestion: refer to attributes and classes, prefix with ~ to have an object name and not the full path.
What do you think? Something like this:
"the :attr:\`~pulpcore.app.models.Task.error\` attribute of the :class:\`~pulpcore.app.models.Task\` object"